ruleset/decoders/0065-cisco-ios_decoders.xml

```diff
@@ -270,22 +270,3 @@ Details: https://www.cisco.com/c/en/us/td/docs/routers/access/wireless/software/
   <regex>%(\w+)-(\d)-(\w+):</regex>
   <order>cisco.facility, cisco.severity, cisco.mnemonic</order>
 </decoder>
-
```

Comment:
I'm not sure but even though this is a revert shouldn't the copyright header be fixed to the latest one since it's still going to be merged into 4.3?

Reply:
with a revert like this, let's verify year is ok, we will rework this for 4.4.0